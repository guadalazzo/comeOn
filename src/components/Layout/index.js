import './styles.scss';
import React from 'react';

const Layout = ({ children }) => {
    return <article className="layout">{children}</article>;
};
export default Layout;
