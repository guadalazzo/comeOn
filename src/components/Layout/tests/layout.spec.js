import React from 'react';
import Layout from '../index';
import renderer from 'react-test-renderer';


describe('Test Layout component', () => {
    test('Layout snapshot', () => {
        const component = renderer.create(<Layout>Home</Layout>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
