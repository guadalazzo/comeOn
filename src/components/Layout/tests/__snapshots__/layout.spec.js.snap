// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Layout component Layout snapshot 1`] = `
<article
  className="layout"
>
  Home
</article>
`;
